pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -110,58 +105,96 @@ func (c *Cluster) configureDashboardModules() error {
 	return nil
 }
 
-func (c *Cluster) configureDashboardModuleSettings(daemonID string) (bool, error) {
+// Delete the manager per-daemon configuration. Returns true
+// if all the configuration entries have been delete successfully.
+func (c *Cluster) deleteManagerDaemonConfiguration() bool {
```

Comment:
No, because we didn't have to. Since now we are moving to a different configuration scheme (at component level instead of daemon level) then we are deleting the previous configuration (per daemon) as it has more priority.